statistics/feedback.go

```diff
@@ -1164,7 +1164,7 @@ func (q *QueryFeedback) dumpRangeFeedback(h *Handle, ran *ranger.Range, rangeCou
 			ran.HighVal[0] = getMaxValue(q.hist.tp)
 		}
 	}
-	ranges := q.hist.SplitRange([]*ranger.Range{ran})
+	ranges := q.hist.SplitRange(nil, []*ranger.Range{ran})
```

Comment:
Should we use the `sc` in line 1145?

Reply:
index value in histogram are stores as bytes. So use `nil` is okay?

Reply:
It may also be column, and we support column types like `timestamp`, so `nil` may not suit here.

Reply:
@lamxTyler fixed